src/utils/index.ts

```diff
@@ -1,5 +1,16 @@
+import axios, { AxiosError } from "axios";
 import { GELATO_ADDRESSES } from "../constants";
 
 export function isGelatoOpsSupported(chainId: number): boolean {
   return Boolean(GELATO_ADDRESSES[chainId]);
 }
+
+export function errorMessage(err: Error | AxiosError) {
```

Comment:
I think we probably have one similar function on the GelatoOpsSdk lib 
please check and update to use same error handling function everywhere

Reply:
[53c69fd](https://github.com/gelatodigital/ops-sdk/pull/25/commits/53c69fd513b5505ad6dfdccac8ee273d4e8f7c06)